packages/react-core/src/components/MultipleFileUpload/examples/MultipleFileUploadBasic.tsx

```diff
@@ -68,7 +69,7 @@ export const MultipleFileUploadBasic: React.FunctionComponent = () => {
   };
 
   // callback that will be called by the react dropzone with the newly dropped file objects
-  const handleFileDrop = (droppedFiles: File[]) => {
+  const handleFileDrop = (_event: DropEvent, droppedFiles: File[]) => {
```

Comment:
Not a blocker for m since this is a limitation of the documentation framework.  `DropEvent` is not documented so the consumer doe not know what that is.    I also notice DropzonOptions is not documented either.
Maybe we can add descriptions of both [here](https://patternfly-react-pr-8995.surge.sh/components/file-upload/multiple-file-upload#multiplefileupload) until the doc framework is updated.